packages/openapi-ts/src/fixSchema.ts

```diff
@@ -0,0 +1,50 @@
+import type { OpenApi } from './openApi/types';
+import type { Config } from './types/config';
+
+interface FixSchemaArgs {
+  data: unknown;
+  fix?: Config['input']['fix'];
+}
+
+export const fixSchema = ({ data, fix }: FixSchemaArgs) => {
+  if (!data || !fix?.schema) {
+    return;
+  }
+
+  const spec = data as OpenApi.V2_0_X | OpenApi.V3_0_X | OpenApi.V3_1_X;
+  const schema = fix.schema;
+
+  const processSchemas = (schemas: Record<string, any>) => {
+    for (const key in schemas) {
```

Comment:
It would be more efficient to loop through `fix` objects as they'll be always smaller (or equal) to schemas objects

Reply:
That's a great point! Thanks for catching that.